editor/scene_tree_dock.cpp

```diff
@@ -84,6 +84,14 @@ void SceneTreeDock::input(const Ref<InputEvent> &p_event) {
 			pending_click_select = nullptr;
 		}
 	}
+
+	Ref<InputEventKey> k = p_event;
+
+	if (k.is_valid() && k->is_pressed()) {
```

Comment:
This should be either in `unhandled_key_input()` or `gui_input()` (preferred), otherwise it's triggered globally disregarding current focus.

FileSystem dock does this correctly.